etl/PAIMS_PIMS/PAIMS_PIMS/PAIMS 1.conmgr

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0"?>
+<DTS:ConnectionManager xmlns:DTS="www.microsoft.com/SqlServer/Dts"
+  DTS:DelayValidation="True"
+  DTS:ObjectName="PAIMS 1"
+  DTS:DTSID="{2CA26938-B681-4221-A68A-6E558882CE8E}"
+  DTS:CreationName="OLEDB">
+  <DTS:ObjectData>
+    <DTS:ConnectionManager
+      DTS:ConnectRetryCount="1"
+      DTS:ConnectRetryInterval="5"
+      DTS:ConnectionString="Data Source=musket.th.gov.bc.ca:20208/PRD11;User ID=DOFILTEA;Provider=OraOLEDB.Oracle.1;" />
```

Comment:
@devinleighsmith  - I don't remember were we landed on storing SQL connection details in git. Is this okay?

Reply:
yes, as long as the sensitive portions are omitted this is fine.